### ISSUE TITLE
Please give full recommended command line options for `cargo doc` in the section of the book describing vlan-switch

### ISSUE THREAD

**Issue by jafingerhut** [oxidecomputer/p4#57]
It mentions that one can use `cargo doc` to generate documentation.  For those who are not familiar with the Rust tools, it would be good to suggest a full command line with all options for this command that produces successful results.

For example, I tried poking around and tried `cargo doc --bin`.  This _did_ generate these files:

```bash
$ cargo doc --bins 
    Finished dev [unoptimized + debuginfo] target(s) in 0.05s
   Generated /home/andy/p4/target/doc/hello_world/index.html
   Generated /home/andy/p4/target/doc/vlan_switch/index.html
```

The book section on the vlan-switch program implies that the generated documentation should document all of the control plane API functions, e.g. the functions with names beginning with `add` or `remove`.  I did not see any functions with those names in the html files mentioned above, but perhaps I am using the wrong command line options to do so.

**Comment by rcgoodfellow** [comment]
Hey Andy. Using `cargo doc` without any options should work. I believe you'll find the methods you're looking for in the documentation for the `main_pipeline` type. 

I've expanded the section in the book on using `cargo doc` to include this information here.

- #60

LMK if that works.

**Comment by jafingerhut** [comment]
Yes, that definitely helps to find the control plane API functions.  Thanks!

**Comment by jafingerhut** [comment]
Fixed with #60.  Closing.
